src/sign-typed-data.ts

```diff
@@ -159,7 +161,7 @@ function encodeField(
       'bytes32',
       version === SignTypedDataVersion.V4 && value == null // eslint-disable-line no-eq-null
         ? '0x0000000000000000000000000000000000000000000000000000000000000000'
-        : keccak(encodeData(type, value, types, version)),
+        : toBuffer(keccak256(encodeData(type, value, types, version))),
```

Comment:
is there a more explicit conversion utility? bigint to buffer or whatever?

Reply:
`keccak256` returns a `Uint8array` so I've swapped this out for `arrToBufArr` from `@ethereumjs/util`